### ISSUE TITLE
Error's

### ISSUE THREAD

**Issue by efkanerkmen** [nicknochnack/LipNet#10]
Can u check full code cause I'm taking a lot of error.I get my first error on frames, alignments = data.as_numpy_iterator().next() on 2. Part of the code

**Comment by SushovanHaldar** [comment]
Yes I am also getting this error as it is not able to convert Tensorflow EagerTensor to a numpy iterable object. 
The error is -: AttributeError: 'tensorflow.python.framework.ops.EagerTensor' object has no attribute 'split'

**Comment by ronaktawde** [comment]
I was running this code in google colab on windows machine. I got this error and if you are also using the same then try this solution. 
==> go to --> def load_data function
==> uncomment --> file_name = path.split('/')[-1].split('.')[0]
==> commnet --> # file_name = path.split('\\')[-1].split('.')[0]

then below that replace('//) with ('/') in the following line
tf.convert_to_tensor(test_path).numpy().decode('utf-8').split('/')[-1].split('.')[0] 

followed by resart and run all cell

This has solved the error.
